ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadWork.java

```diff
@@ -136,8 +142,9 @@ public ReplLoadWork(HiveConf hiveConf, String dumpDirectory,
       FileSystem fs = failoverReadyMarker.getFileSystem(hiveConf);
       shouldFailover = hiveConf.getBoolVar(HiveConf.ConfVars.HIVE_REPL_FAILOVER_START)
               && fs.exists(failoverReadyMarker);
-      isFailover =
-          checkFileExists(new Path(dumpDirectory).getParent(), hiveConf, OptimisedBootstrapUtils.EVENT_ACK_FILE);
+      isFirstFailover = checkFileExists(new Path(dumpDirectory).getParent(), hiveConf, EVENT_ACK_FILE);
```

Comment:
nit: Could create and re-use the Path object. e.g
Path dumpDirParent = new Path(dumpDirectory).getParent();

Reply:
done